tools/fapi/tss2_changeauth.c

```diff
@@ -0,0 +1,69 @@
+/* SPDX-License-Identifier: BSD-2 */
+/*******************************************************************************
+ * Copyright 2019, Fraunhofer SIT sponsored by Infineon Technologies AG
+ * All rights reserved.
+ ******************************************************************************/
```

Comment:
no copy right clauses in any file, this and throughout. Those statements are superfluous.

Reply:
For those that want these, they can place them in the LICENSE file, which you already have one.

Reply:
willdo